src/use_case/vote_out/VoteOutInteractor.java

```diff
@@ -2,19 +2,20 @@
 
 import entity.Game;
 import entity.PromptGenerator;
-import use_case.ChatAPIAccessInterface;
+import use_case.data_access_interface.ChatAPIAccessInterface;
+import use_case.data_access_interface.PromptGameDataAccessInterface;
 
 public class VoteOutInteractor implements VoteOutInputBoundary {
-    private final VoteOutDataAccessInterface gameDataAccessObject;
-    private final ChatAPIAccessInterface storyDataAccessObject;
+    private final PromptGameDataAccessInterface gameDataAccessObject;
+    private final ChatAPIAccessInterface gptDataAccessObject;
     private final VoteOutOutputBoundary userPresenter;
     private final Game game;
     private final PromptGenerator promptGenerator;
 
 
-    public VoteOutInteractor(VoteOutDataAccessInterface gameDataAccessObject, ChatAPIAccessInterface storyDataAccessObject, VoteOutOutputBoundary userPresenter) {
+    public VoteOutInteractor(PromptGameDataAccessInterface gameDataAccessObject, ChatAPIAccessInterface gptDataAccessObject, VoteOutOutputBoundary userPresenter) {
         this.gameDataAccessObject = gameDataAccessObject;
-        this.storyDataAccessObject = storyDataAccessObject;
+        this.gptDataAccessObject = gptDataAccessObject;
         this.userPresenter = userPresenter;
         // Get the game
         game = gameDataAccessObject.getGame();
```

Comment:
From what I can see PromptGameDataAccess does not have a getGame() method, correct me if I'm wrong but it seems to be a refactoring mistake?

Reply:
it automatically has the methods in the class it's extending, check my interactor it works